src/js/components/uploadFabsFile/UploadFabsFileValidation.jsx

```diff
@@ -228,42 +251,42 @@ export class UploadFabsFileValidation extends React.Component {
         // check if the link is already signed
         if (this.state.signInProgress) {
             // sign is in progress, do nothing
-
         }
-        else if (this.state.signedUrl !== "") {
+        else if (this.state.signedUrl !== '') {
             // it is signed, open immediately
             this.openReport();
         }
         else {
             // not signed yet, sign
-            this.setState({
-                signInProgress: true
-            }, () => {
-                this.signReport(item);
-            });
+            this.setState(
+                {
+                    signInProgress: true
+                },
+                () => {
+                    this.signReport(item);
+                }
+            );
         }
     }
 
     submitFabs() {
-        this.setState({ submit: false, published: 'publishing', showPublish: false },
-            () => {
-                UploadHelper.submitFabs({ submission_id: this.props.submission.id })
-                    .then(() => {
-                        this.checkFile(this.props.submission.id);
-                    })
-                    .catch((error) => {
-                        if (error.httpStatus === 400) {
-                            this.setState({ error: 1, error_message: error.message, published: 'unpublished' });
-                        }
-                        else if (error.httpStatus === 500) {
-                            this.setState({ error: 4, error_message: error.message, published: 'unpublished' });
-                        }
-                        else {
-                            this.setState({ error: 1, error_message: error.message, published: 'unpublished' });
-                        }
-                    });
-            }
-        );
+        this.setState({ submit: false, published: 'publishing', showPublish: false }, () => {
+            UploadHelper.submitFabs({ submission_id: this.props.submission.id })
+                .then(() => {
+                    this.checkFile(this.props.submission.id);
+                })
+                .catch((error) => {
+                    if (error.httpStatus === 400) {
+                        this.setState({ error: 1, error_message: error.message, published: 'unpublished' });
+                    }
+                    else if (error.httpStatus === 500) {
+                        this.setState({ error: 4, error_message: error.message, published: 'unpublished' });
+                    }
+                    else {
```

Comment:
Since we're already cleaning this up, isn't this the same as the first `if` statement? Could we just do a check for `500` and then else without the other if?